plugins/woocommerce/client/admin/client/homescreen/layout.js

```diff
@@ -163,7 +168,44 @@ export const Layout = ( {
 				{ isDashboardShown ? renderColumns() : renderTaskList() }
 				{ shouldShowMobileAppModal && <MobileAppModal /> }
 			</div>
-		</>
+			<Card className="woocommerce-customize-store-browse-marketplace">
```

Comment:
Code (and its corresponding style) shamelessly copied from https://github.com/woocommerce/woocommerce/pull/55428 :)